azure/resource_groups/app/parameters/development.template.json

```diff
@@ -149,7 +149,7 @@
       }
     },
     "LOGSTASH_PORT": {
-      "value": 17002
```

Comment:
Looks like we're changing from TCP-SSL. Why did we make this change? I think it would be good to put a bit of explanation in the commit message.

Reply:
Also, we should get rid of the `logstash-logger` gem.

Reply:
Thanks Lawrence. I've updated the commit message and added a commit that removes the `logstash-logger` gem.